CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreeCore
+  * Fix bug where analytics wouldn't send on `BTAPIClient` instantiation on button click
+  * Update `endpoint` syntax sent to FPTI for 3D Secure and Venmo flows
```

Comment:
Ah shoot, totally didn't catch this from your release last week, thanks for fixing it for me!